plugin/ochttp/stats.go

```diff
@@ -22,25 +22,33 @@ import (
 
 // The following client HTTP measures are supported for use in custom views:
 var (
-	ClientRequestCount, _  = stats.Int64("opencensus.io/http/client/request_count", "Number of HTTP requests started", stats.UnitNone)
-	ClientRequestBytes, _  = stats.Int64("opencensus.io/http/client/request_bytes", "HTTP request body size if set as ContentLength (uncompressed)", stats.UnitBytes)
-	ClientResponseBytes, _ = stats.Int64("opencensus.io/http/client/response_bytes", "HTTP response body size (uncompressed)", stats.UnitBytes)
-	ClientLatency, _       = stats.Float64("opencensus.io/http/client/latency", "End-to-end latency", stats.UnitMilliseconds)
+	ClientRequestCount  = stats.MustInt64("opencensus.io/http/client/request_count", "Number of HTTP requests started", stats.UnitNone)
```

Comment:
I don't think we should increase the API surface with stats.MustX and tag.MustX.

For hardcoded values, it is ok to avoid the error. We do it all the time for the http.NewRequest call.

You will often see, error is avoided for hardcoded values that are not going to fail:

```
req, err := http.NewRequest("GET", "http://example.com", nil)
```

If we don't do that, we are almost deprecating NewX functions because they are in practice useful in rare cases, e.g. if you are dynamically creating keys dynamically.

Reply:
I concur about not increasing the API surface layer; I had suggested those functions as internal helper functions.

@rakyll as for the example
> You will often see, error is avoided for hardcoded values that are not going to fail:
> req, err := http.NewRequest("GET", "http://example.com", nil)
req, err := http.NewRequest("GET", uri, nil)

requires an error because there are lots of cases where you'll pass in a URL that isn't known at compile time and if that URL fails to parse you'll get an error for example when downloading URLs constructed at runtime, in a proxy, on a backend etc.

Reply:
Btw, what I meant above was actually:

```
req, _ := http.NewRequest("GET", "http://example.com", nil)
```

Reply:
Gotcha gotcha, in deed, thanks.